sdk/communication/Azure.Communication.Calling.Server/src/Generated/AzureCommunicationCallingServerServiceModelFactory.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.Communication.Calling.Server
+{
+    /// <summary> Model factory for read-only models. </summary>
+    public static partial class AzureCommunicationCallingServerServiceModelFactory
+    {
+        /// <summary> Initializes new instance of CreateCallResponse class. </summary>
+        /// <param name="callLegId"> Call leg id of the call. </param>
+        /// <returns> A new <see cref="Server.CreateCallResponse"/> instance for mocking. </returns>
+        public static CreateCallResponse CreateCallResponse(string callLegId = default)
+        {
```

Comment:
nit: you can use expression body for all of these one-liner methods

Reply:
so far we don't have idea on how to use them.

Reply:
See [here ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members )for more on expression bodied members. You can transform it to this syntax: 

`public static CreateCallResponse CreateCallResponse(string callLegId = default) -> new CreateCallResponse(callLegId);`